rest-api/dao/participant_summary_dao.py

```diff
@@ -201,9 +204,12 @@ def update_from_biobank_stored_samples(self, participant_id=None):
     If participant_id is provided, only that participant will have their summary updated."""
     baseline_tests_sql, baseline_tests_params = get_sql_and_params_for_array(
         config.getSettingList(config.BASELINE_SAMPLE_TEST_CODES), 'baseline')
+    self._back_fix_new_codes(baseline_tests_params)
     dna_tests_sql, dna_tests_params = get_sql_and_params_for_array(
         config.getSettingList(config.DNA_SAMPLE_TEST_CODES), 'dna')
     sample_sql, sample_params = _get_sample_sql_and_params()
+    self._back_fix_new_codes(sample_params)
```

Comment:
Similarly we don't need to call this here. (Because of what I mentioned above, and also because 1PST8, etc. aren't in the DNA sample list.)